README.md

```diff
@@ -12,16 +12,16 @@ stakeholders can be confident about the direction the project is evolving in.
 
 ## How to submit an RFC
 
-1. Copy `template.md` into `text/YYYY-MM-DD-my-feature.md`.
+1. Copy `template.md` into `text/PRID-my-feature.md`.
```

Comment:
This change seems unrelated to the rest of this PR

Reply:
I will revert these changes and let's discuss on another PR. Briefly I think it is easy for someone pronounces/refers `RFC-PRID` instead of `RFC-YYYY-MM-DD` or `RFC-my-feature`.